src/Scenes/CheckLocation.fs

```diff
@@ -20,7 +20,7 @@ type Status =
 | Changed
 | Error of string
 
-type Model =
+type [<Pojo>] Model =
```

Comment:
Ah, ok. Now I remember: I think Elmish actually wraps the model in a Pojo and uses F#/Fable comparison to see if the view needs to be updated. So you're right, probably it shouldn't be needed, I'll check.